config/locales/image.es.yml

```diff
@@ -0,0 +1,35 @@
+en:
+  hyrax:
+    icons:
+      image:     'fa fa-picture-o'
+    select_type:
+      image:
+        name:               "Imagen"
+        description:        "La imagen de una o varias páginas funciona."
+  simple_form:
+    labels:
+      image:
+        based_near:       "Ubicación"
+        description:      "Resumen o Sumario"
+        keyword:          "Palabra Clave"
+        date_created:     "Fecha de creación"
+        related_url:      "URL relacionada"
+        files:            "Cargar una nueva versión de este archivo desde tu computadora"
+        collection_ids:   "Agregar como miembro de la colección"
+        admin_set_id:     "Agregar como miembro del conjunto administrativo"
+    hints:
+      image:
+        based_near: "Nombre de un lugar relacionado con la imagen, tal como el sitio de publicación, o ciudad, estado, o país, que esté relacionado con los contenidos de la imagen. Llamados a través del servicio de <a href='http://www.geonames.org'>GeoNames</a>."
```

Comment:
Should we set default hints in hyrax? https://github.com/projecthydra-labs/hyrax/blob/master/config/locales/hyrax.es.yml#L628